Package.swift

```diff
@@ -114,14 +125,19 @@ if let env = env, String(cString: env, encoding: .utf8) == "1" {
                 .headerSearchPath("SentryCrash/Installations"),
                 .headerSearchPath("SentryCrash/Reporting/Filters"),
                 .headerSearchPath("SentryCrash/Reporting/Filters/Tools"),
-                .define("SDK_V9")])
+                .define("SDK_V9")
+            ]
+        )
     ])
 }
 
 let package = Package(
     name: "Sentry",
     platforms: [.iOS(.v15), .macOS(.v12), .tvOS(.v15), .watchOS(.v8)],
     products: products,
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-log", from: "1.6.0")
```

Comment:
`h`: So I think this is going to be a problem as soon as new `swift-log` version becomes available. AFAIK the SPM doesn't support peer-dependencies, so if someone uses `swift-log` with a version not being `1.6.0`, while we depend on it, Xcode will not build because it doesn't support the same dependency with different versions like NPM.

So basically this will lock our SDK users into exactly the `swift-log` version we declare, requiring us to release a new SDK version with every update of `swift-log`, furthermore requiring users to update the Sentry SDK even though they just want a new version of `swift-log`. At this point I do not have a proposal how to fix this other than asking the user to "inject" their `swift-log` version by making it conform to a Sentry-included protocol

Reply:
In that case it should be possible to use `from:` without breaking users. If swift-log releases a new major we might even have to fix our integration and need an SDK update anyways. I am mostly concerned about us requiring a specific version while other SDKs/the App project itself conflicts with it.

We can always downgrade the minimum supported version of swift-log in another SDK release.